lib/storage/simple_storage.js

```diff
@@ -28,13 +28,23 @@ module.exports = function(config) {
 
     if (!config) {
         config = {
-            path: './',
+            path: './'
         };
     }
 
-    var teams_db = new Store(config.path + '/teams', {saveId: 'id'});
-    var users_db = new Store(config.path + '/users', {saveId: 'id'});
-    var channels_db = new Store(config.path + '/channels', {saveId: 'id'});
+    config.path = config.path || './';
```

Comment:
Set default for config.path if not set. This was necessary in case someone failed to include config.path when they passed in config for the new in-memory option.